demos/openshift-install/java-client-installer.sh

```diff
@@ -29,6 +29,33 @@ function validate {
   validate_app keytool
 }
 
+function prepare_input {
+
+  local PARAM_VALUE=$1
+  local PARAM_NAME=$2
+  local DEFAULT_VALUE=$3
+
+  if [ "$PARAM_VALUE" == "" ]; then
+    if [ -n "$DEFAULT_VALUE" ]; then
+      read -p "Please specify the $PARAM_NAME {$DEFAULT_VALUE}:" PARAM_VALUE
```

Comment:
The standard is that for the default values, is that they are written in `[` and `]`

```suggestion
      read -p "Please specify the $PARAM_NAME [$DEFAULT_VALUE]:" PARAM_VALUE
```

Reply:
done